lib/python/qmk/path.py

```diff
@@ -20,6 +24,56 @@ def keymap(keyboard):
     raise NoSuchKeyboardError('Could not find keymaps directory for: %s' % keyboard)
 
 
+def check_directories(*directories, exist=True, dont_raise=False):
```

Comment:
Added this generic helper as something to use to check presence/absence of a directory.